geode-core/src/test/java/org/apache/geode/management/internal/cli/commands/ShowMetricsDUnitTest.java

```diff
@@ -89,16 +80,54 @@ public void run() {
         Cache cache = getCache();
         RegionFactory<Integer, Integer> dataRegionFactory =
             cache.createRegionFactory(RegionShortcut.REPLICATE);
-        Region region = dataRegionFactory.create("REGION1");
+        dataRegionFactory.create("REGION1");
       }
     });
+  }
 
+  private boolean createMBean(int beanType, String regionName, DistributedMember distributedMember,
+      int cacheServerPort) {
+    Cache cache = getCache();
+    ManagementService mgmtService = ManagementService.getManagementService(cache);
+    Object bean = null;
+
+    switch (beanType) {
+      case 1:
+        bean = mgmtService.getDistributedSystemMXBean();
+        break;
+      case 2:
+        bean = mgmtService.getDistributedRegionMXBean("/" + regionName);
+        break;
+      case 3:
+        ObjectName memberMBeanName = mgmtService.getMemberMBeanName(distributedMember);
+        bean = mgmtService.getMBeanInstance(memberMBeanName, MemberMXBean.class);
+        break;
+      case 4:
+        ObjectName regionMBeanName =
+            mgmtService.getRegionMBeanName(distributedMember, "/" + regionName);
+        bean = mgmtService.getMBeanInstance(regionMBeanName, RegionMXBean.class);
+        break;
+      case 5:
+        ObjectName csMxBeanName =
+            mgmtService.getCacheServerMBeanName(cacheServerPort, distributedMember);
+        bean = mgmtService.getMBeanInstance(csMxBeanName, CacheServerMXBean.class);
+        break;
+    }
+
+    return bean != null;
+  }
+
+  /**
+   * tests the default version of "show metrics"
+   */
+  @Test
+  public void testShowMetricsDefault() {
+    systemSetUp();
     SerializableCallable showMetricCmd = new SerializableCallable() {
 
       @Override
       public Object call() throws Exception {
-        WaitCriterion wc = createMBeanWaitCriterion(1, "", null, 0);
-        waitForCriterion(wc, 5000, 500, true);
+        await().atMost(5, SECONDS).until(() -> createMBean(1, "", null, 0));
```

Comment:
I think it would be safer to set a longer maximum wait time (perhaps 2 minutes?)  in case this runs during a GC pause on a slow machine.  The same comment applies to some other await() calls in the same file.